### ISSUE TITLE
Weird validation acc/loss  

### ISSUE THREAD

**Issue by terrychenism** [apache/incubator-mxnet#2282]
I have follow new resnet cifar10 symbol to create new resnet for imagenet symbol. After 30 epoch, validation acc is 71%+ (almost same as paper, https://github.com/KaimingHe/deep-residual-networks ), then gradually decrease to 69% at 48 epoch. Attach is the validation log.
![qq 20160529235525](https://cloud.githubusercontent.com/assets/3353054/15641922/ddbd6008-25f8-11e6-8fb8-e8ea6ea3c973.jpg) 

Is there anyone who has happen to the same problem for training resnet on imagent?

**Comment by piiswrong** [comment]
It's overfitting. You need more data augumentations. The facebook version used these: https://github.com/facebook/fb.resnet.torch/blob/master/datasets/transforms.lua

**Comment by terrychenism** [comment]
Yes, I forgot scale augment. In my caffe  resnet experiment, I use full image to do scale augment. Thanks a lot Eric!

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
